app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityPresenter.kt

```diff
@@ -75,7 +75,7 @@ class AdministratorControlsActivityPresenter @Inject constructor(
   fun loadProfileList() {
     lastLoadedFragment = PROFILE_LIST_FRAGMENT
     getAdministratorControlsFragment()!!.setSelectedFragment(lastLoadedFragment)
-    activity.supportFragmentManager.beginTransaction().add(
+    activity.supportFragmentManager.beginTransaction().replace(
```

Comment:
why these changes?

Reply:
Also check this comment too - https://github.com/oppia/oppia-android/pull/3185#discussion_r631535910

Reply:
I've done these changes because as I already mentioned that the application is loading the same fragment over the previously opened same fragmet if we click the button twice in multipane view. Also clarified this problem in this comment: https://github.com/oppia/oppia-android/pull/3185#issuecomment-841169003